dropshot_endpoint/Cargo.toml

```diff
@@ -12,6 +15,6 @@ proc-macro2 = "1"
 quote = "1"
 serde = { version = "1.0", features = [ "derive" ] }
 serde_derive_internals = "0.25"
-serde_tokenstream = { git = "https://github.com/oxidecomputer/serde_tokenstream" }
+serde_tokenstream = { version = "0.1.0", git = "https://github.com/oxidecomputer/serde_tokenstream" }
```

Comment:
I could imagine just having the version here and delete the git part; I don't think there's a strong argument either way. If it's just version then it will make development a little tricky for interlocking changes, but then we haven't had those in a while. If we have git in there too we'll just need to remember to publish that version if we want to update dropshot which might be kind of a pain.

Reply:
According to the cargo output, the git version is removed before it gets published.  From a previous commit where I didn't have the "version" field:

```
dropshot_endpoint $ cargo publish --dry-run
    Updating crates.io index
error: all dependencies must have a version specified when publishing.
dependency `serde_tokenstream` does not specify a version
Note: The published dependency will use the version from crates.io,
the `git` specification will be removed from the dependency declaration.
dropshot_endpoint $ 
```


So I don't think having the "git" field there makes things much easier than if you added your own "patch" section to Cargo.toml or replace the "version" with a "git" URL while you're working on it.  Given that, I'm tempted to just drop the git part.  What do you think?

Reply:
I would drop the git part.

Reply:
yes, let's drop it. If it turns out there are a bunch of interlocking changes then we can reconsider.

Reply:
Fixed in 584593e8a05e7a80e9b1de1d6dd7022883156dab.